packages/design-system-docs/src/pages/components/Table/_Table.docs.scss

```diff
@@ -76,23 +75,13 @@ Style guide: components.table.react-tablerow
 */
```

Comment:
I actually wonder if we can remove the `@react-props` for `TableCaption`, `TableHead`, and `TableBody`, as they are all basically supposed to be used like HTML.
https://github.com/CMSgov/design-system/pull/796/files#diff-c408290eec1f1b2fbb29243804f68058L63

The only important info in the react prop docs for those components right now is what each component usually contains as its children, and we could easily move that to the component description. 

Before:
<img width="767" alt="Screen Shot 2020-09-02 at 2 05 49 PM" src="https://user-images.githubusercontent.com/5424713/92025733-96ee6480-ed25-11ea-8443-3ea056d7a718.png">
After:
<img width="644" alt="Screen Shot 2020-09-02 at 2 06 46 PM" src="https://user-images.githubusercontent.com/5424713/92025739-9950be80-ed25-11ea-8b28-11df349d9824.png">

Reply:
That does keep the docs leaner to capture the info in description only.

Reply:
There's an issue with our doc generation script if I remove the props, the `view source file` will not sure up. But if i just @hide-prop all the props, the doc site will just display the header without content. 
To avoid having an incomplete doc site, I'll keep this present layout and add this item to the list of changes we can look into.
<img width="793" alt="Screen Shot 2020-09-02 at 4 45 21 PM" src="https://user-images.githubusercontent.com/20322880/92035073-2a329480-ed3c-11ea-8196-afb4975659be.png">

Reply:
@nichia Maybe we could write another ticket for this work?